### ISSUE TITLE
assimp - aiGetMaterialTexture cuts first chars of texture Path

### ISSUE THREAD

**Issue by LordAlbion** [LWJGL/lwjgl3#516]
### Environment 

* LWJGL version: 3.2.4 
* LWJGL build #: 2
* Java version: jdk-13.0.1
* Platform: `Windows`
* Module: `assimp`

### Description

Assimp retrieves name of the texture cut by first 4 first chars.

insted of "123.jpg"
i get "jpg    " - looks like the buffer is moved by 4 chars more than should be

Some issue with encoding or with buffer read length

Example material and code to retrieve data:

For material:
```
# Blender MTL File: 'None'
# Material Count: 1

newmtl Material.001
Ns 500.039216
Ka 0.000000 0.000000 0.000000
Kd 1.000000 1.000000 1.000000
Ks 1.000000 1.000000 1.000000
Ke 0.000000 1.200000 0.000000
Ni 0.500000
d 1.000000
illum 2
map_Kd 123.jpg
```

For fallowing code
```
AIString mTexPath = AIString.calloc();
			aiGetMaterialTexture(mMaterial, 1, 0, mTexPath, (IntBuffer) null, null, null, null, null, null);
			
			System.out.println(mTexPath.dataString());
```

result: "jpg    "
should be "123.jpg"

**Comment by LordAlbion** [comment]
could this be issue with pointer struct?
public abstract class Struct extends Pointer.Default {

    **protected static final int DEFAULT_PACK_ALIGNMENT = Platform.get() == Platform.WINDOWS ? 8 : 0x4000_0000;**
    protected static final int DEFAULT_ALIGN_AS       = 0;


similar definition in Pointer interface 

    /** The value of {@code sizeof(long)} for the current platform. */
    int CLONG_SIZE  = POINTER_SIZE == 8 && Platform.get() == Platform.WINDOWS ? 4 : POINTER_SIZE;

**Comment by Spasi** [comment]
Hey @LordAlbion,

No, this is a simple type mismatch in LWJGL's definition for `aiString`. It was caused by [this change](https://github.com/assimp/assimp/commit/69087abc56237d4fdf6557a47602583b7fe733ac#diff-bce844cf44c48de5ffc33753711a9b36R387), which I missed the last time the assimp bindings were updated. It will be fixed in the next 3.2.4 snapshot.

**Comment by LordAlbion** [comment]
Thanks
